app/bingey-api/models/user.js

```diff
@@ -0,0 +1,52 @@
+const { Schema, model } = require('mongoose');
+
+const userSchema = new Schema({
+    _id:{
+        type: String,
+        required: true,
+    },
+    name: {
+        type: String,
+        required: true,
+    },
+    password: {
+        type: String,
+        required: true,
+    },
+    registrationDate: {
+        type: Date,
+        required: true,
+    },
+    watchlists: [
+        {
+            watchlistId: {
+                type: Schema.Types.ObjectId,
+                ref: 'Watchlist',
+            },
+        }
+    ],
+    
+	recommended: [
+		{
+            titleId: {
+                type: Schema.Types.ObjectId,
+                ref: 'Titles',
```

Comment:
Mongoose has backward-compatibility breaking quirk with pluralization
```suggestion
                ref: 'Title',
```